sdk/spring/azure-spring-boot/src/test/java/com/azure/spring/autoconfigure/b2c/AADB2CConstants.java

```diff
@@ -8,6 +8,20 @@
 
 public class AADB2CConstants {
 
+    private static final String AUTHENTICATE_ADDITIONAL_PARAMETERS = ".authenticate-additional-parameters";
+
+    private static final String AUTHENTICATE_ADDITIONAL_PARAMETERS_LOGIN_HINT = AUTHENTICATE_ADDITIONAL_PARAMETERS + ".login-hint";
```

Comment:
The following 2 style is better:

1. AUTHENTICATE_ADDITIONAL_PARAMETERS_LOGIN_HINT = AUTHENTICATE_ADDITIONAL_PARAMETERS + "." + PROMPT
2. AUTHENTICATE_ADDITIONAL_PARAMETERS_LOGIN_HINT = ".authenticate-additional-parameters.prompt".

I prefer 2.